homework8_concurrency+io/src/Person.java

```diff
@@ -0,0 +1,98 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Person {
+    String name;
+    LocalDate birthdate;
+    Sex s;
+    List<Person> lista = new ArrayList<>();
+
+    public Person(String name, LocalDate birthdate, Sex s) {
+        this.name = name;
+        this.birthdate = birthdate;
+        this.s = s;
+    }
+
+    public Person() {
+    }
+
+    public void createPerson() throws IOException {
+        try (BufferedReader br1 = new BufferedReader(new FileReader("homework8_concurrency+io/src/file1.txt"))) {
```

Comment:
Citirea din fisier ar fi trebuit implementata in alta clasa, nu in clasa `Person`.